db/migrate/20150422234627_create_userjobs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserjobs < ActiveRecord::Migration
+  def change
+    create_table :userjobs do |t|
```

Comment:
Maybe call this LikedJobs (in-line with HiddenJobs) instead of UserJobs, seems more descriptive of the tables real purpose since a user doesn't really have jobs, they like them.